app/views/obituaries/show.html.erb

```diff
@@ -1,17 +1,36 @@
-<div class="obituary-image">
-  <%= image_tag(@obituary.image_url) rescue image_tag 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg' %>
-</div>
-<div class="obituary-show">
-  <h1><%="#{@obituary.first_name} #{@obituary.last_name}"%></h1>
+<!-- Page Content -->
+<div class="container", style= "background-color:white;">
+
+  <!-- Portfolio Item Heading -->
+  <h1 class="my-4"><%="#{@obituary.first_name} #{@obituary.last_name}"%></h1>
+
+  <!-- Portfolio Item Row -->
+  <div class="row">
 
-  <h2> Age: <%= @obituary.age %> </h2>
-  <h2> City: <%= @obituary.city %> </h2>
-  <h2> State: <%= @obituary.state %> </h2>
+    <div class="col-md-4" id="obituary-image">
+      <%= image_tag(@obituary.image_url) rescue image_tag 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg', {class: "img-fluid"} %>
+    </div>
 
-  <p><%= @obituary.description %></p>
+    <div class="col-md-6">
+      <h3 class="my-3">Obituary</h3>
+      <p><%= @obituary.description %></p>
+      <h3 class="my-3">Details</h3>
+      <ul>
+        <li>Age: <%= @obituary.age %></li>
+        <li>City: <%= @obituary.city %></li>
+        <li>State: <%= @obituary.state %></li>
+        <li>Covid: <%= @obituary.covid %></li>
+      </ul>
+      <br>
+    </div>
 
-  <% if current_user && current_user.has_obituary?(@obituary) %>
-    <p><%= link_to 'Edit', edit_obituary_path(@obituary.id) %></p>
-    <p><%= link_to 'Delete', "/obituaries/#{@obituary.id}", method: :delete %></p>
-  <% end %>
+  </div>
 </div>
+<!-- /.container -->
+
+<% if current_user && current_user.has_obituary?(@obituary) %>
+  <p><%= link_to 'Edit', edit_obituary_path(@obituary.id) %></p>
+  <p><%= link_to 'Delete', "/obituaries/#{@obituary.id}", method: :delete %></p>
+<% end %>
+
+<%# What we display here will be dictated by whether it comes from NYT or is created by a user. User created content will have more information, and link will be optional. NYT will likely have publication date, source, link, and name (not separated into first and last) %>
```

Comment:
Note to self, but also to others...Can you tell I spent the last 24 hours brainstorming how NYT data and user generated data will diverge and how we can fold it all together? TBD if we will make a different show page for NYT obits, how that will work, and what it will display.

Reply:
I'll do a brainstorm myself this morning!